### DISCUSSION TITLE
Pulling prompts by tag?

### DISCUSSION THREAD

**Discussion by mayajosifo** [langfuse/langfuse#3099] [Support]
Is there a way to fetch all production prompts from Langfuse? For instance, if I add a production prompt through the UI, would it be possible to have my program pull it in without having to manually add in the prompt name?

**Comment by marcklingen** [comment]
Yes, you can pull all prompts via the api: https://api.reference.langfuse.com/

**Comment by utkarsh-fileread** [comment]
Is there support for pulling by tag on the Python SDK?
